environment.yml

```diff
@@ -0,0 +1,13 @@
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - wheel
+  - numcodecs
+  - numpy
+  - pip
+  - pip:
+    - asciitree
+    - fasteners
+    - pytest
+    - setuptools_scm
```

Comment:
We might also consider pinning to the minimum supported version of packages

https://github.com/zarr-developers/zarr-python/blob/5c0ef7ff56e957033bde3d6c57a485efb0490b52/setup.py#L9-L14

to make sure that Zarr works both when certain optional dependencies aren't installed, but also when using, for example, `numpy=1.7`. 

(not sure if there's another CI build which already tests this -- feel free to ignore me if there is 🙂 )

Reply:
Pushed.